.github/workflows/_build-wheel-release-upload.yml

```diff
@@ -13,6 +13,11 @@ on:
         default: false
         required: false
         type: boolean
+      github_admin_username:
+        description: 'GitHub admin authorized for GitHub/PyPI release'
+        required: false
+        default: 'sbillinge'
```

Comment:
Let's remove this default.  We will have to update (like a recut) all the projects but it will be quick and this can avoid problems if people actually do start to use this (and we kind of hope they do!)

Reply:
done